docs/source/examples/petstore/petstore.all.rego.compiled

```diff
@@ -68,18 +74,23 @@ allow {
     seal_list_contains(input.subject.groups, `customers`)
     input.verb == `buy`
     re_match(`petstore.pet`, input.type)
-input.status == "available"
+    input.status == "available"
 }
 allow {
     seal_list_contains(input.subject.groups, `breeders_maltese`)
     input.verb == `buy`
     re_match(`petstore.pet`, input.type)
-input.status == "reserved"
-input.breed == "maltese"
+    input.status == "reserved"
+    input.breed == "maltese"
 }
 
 # rego functions defined by seal
 
+# Helper to get the token payload.
+seal_subject = payload {
```

Comment:
I think this is fine for now, but this would be inserted for every policy that is compiled, so this would be evaluated in each policy.  In a future optimization step, I think we should move this into its own rego package which all other compiled policies would import.

Reply:
please create a seal issue and define the optimization